OWML.ModHelper.Logging/Output.cs

```diff
@@ -0,0 +1,44 @@
+﻿using OWML.Common;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace OWML.ModHelper.Logging
+{
+    public static class Output
+    {
+        public static IModConsole OwmlOutput { get; private set; }
+        public static event Action<IModManifest, string> OnWrite;
+
+        public static IModConsole CreateOutput(IOwmlConfig owmlConfig, IModLogger logger, IModManifest manifest)
```

Comment:
This sure looks like it could've been a constructor :)

Reply:
I guess the naming is confusing? It's actually like a CreateModConsole. I don't want to instantiate Output.cs. Some naming changes could help with this. Ideally I'd want to rename IModConsole to something else, but I guess that would be problematic with reverse compatibility.